### PR TITLE
[browser] doc node SIMD+EH, dev tools

### PR DESCRIPTION
**Pull request by pavelsavara**

### CODE REVIEW

File: src/mono/wasm/features.md [hunk 1]
```diff
@@ -40,11 +40,15 @@ It can be enabled with `<WasmEnableSIMD>true</WasmEnableSIMD>` and disabled with
 
 For more information on this feature, see [SIMD.md](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md).
 
+Older versions of NodeJS hosts may need `--experimental-wasm-simd` command line option.
```
Comment (ilonatommy) [inline src/mono/wasm/features.md:43]: Can we specify what "older" means here?

**Review by ilonatommy — APPROVED** [review]

File: src/mono/wasm/features.md (hunk 1 above)
Comment (pavelsavara) [reply src/mono/wasm/features.md:43]: I can't find it. @radekdoulik ?

File: src/mono/wasm/features.md [hunk 2]
```diff
@@ -312,12 +316,82 @@ it may be better if you re/write your logic in Web native technologies like HTML
...
+
+You can add following elements in your .csproj
+```xml
+<PropertyGroup>
+  <WasmNativeDebugSymbols>true</WasmNativeDebugSymbols>
+  <WasmNativeStrip>false</WasmNativeStrip>
+</PropertyGroup>
+```
+
+See also DWARF [WASM debugging](https://developer.chrome.com/blog/wasm-debugging-2020/) in Chrome.
```
Comment (ilonatommy) [inline src/mono/wasm/features.md:350]: When we're talking about DWARF we could add link to this doc: https://github.com/dotnet/runtime/blob/main/src/mono/wasm/debugger/debugger.md

File: src/mono/wasm/features.md [hunk 3]
```diff
@@ -312,12 +316,82 @@ it may be better if you re/write your logic in Web native technologies like HTML
...
+You could also debug the C# code using our integration with browser dev tools or Visual Studio.
+See detailed [documentation](https://learn.microsoft.com/en-us/aspnet/core/blazor/debug)
+
+### Native debug symbols
+
+You can add following elements in your .csproj
+```xml
+<PropertyGroup>
+  <WasmNativeDebugSymbols>true</WasmNativeDebugSymbols>
+  <WasmNativeStrip>false</WasmNativeStrip>
```
Comment (ilonatommy) [inline src/mono/wasm/features.md:346]: Isn't this procedure described in this doc: https://github.com/dotnet/runtime/blob/main/docs/workflow/debugging/mono/wasm-debugging.md?

Comment (pavelsavara) [reply src/mono/wasm/features.md:346]: That's more runtime team facing and low level. My document is more for external consumers. I will link it for details

**Pull request merged**